eslint-config-lib/package.json

```diff
@@ -20,7 +17,10 @@
     "url": "https://github.com/inrupt/javascript-style-configs/issues"
   },
   "homepage": "https://github.com/inrupt/javascript-style-configs#readme",
-  "peerDependencies": {
-    "@inrupt/eslint-config-base": "^0.4.0"
+  "dependencies": {
+    "@inrupt/eslint-config-base": "file:../eslint-config-base",
```

Comment:
Interesting! I'm not familiar with this, how will it behave in the published package?

I guess everything is based on the package-lock there, so everything will be fine. That's cool :)

Reply:
this is handled by the release tooling, which sets the correct versions